controllers/util/storageconsumer.go

```diff
@@ -156,10 +156,10 @@ func (wrapper storageConsumerResourceMapWrapper) SetNfsClientProfileName(name st
 	wrapper.data[nfsClientProfileKey] = name
 }
 
-func GetStorageConsumerDefaultResourceNames(storageConsumerName, fsid string) map[string]string {
+func GetStorageConsumerDefaultResourceNames(storageConsumerName, storageConsumerUid string) map[string]string {
 
-	//TODO: Revisit this to use a different method to calculate the name
-	clientProfileName := CalculateMD5Hash([2]string{fsid, storageConsumerName})
+	//TODO: Revisit this to use a different method to calculate the hash
+	clientProfileName := CalculateMD5Hash([2]string{storageConsumerUid, storageConsumerName})
```

Comment:
Why do we use bot name and uid? you don't even need the variable, just use the uid in place